content/faq/_index.md

```diff
@@ -149,3 +149,53 @@ you need and us making them available for you.
 
 In the mean time we are solving these requests one by one, so [please reach out
 to us](#how-can-i-contact-you).
+
+### A command failed in packit-service: how do I reproduce it locally?
+
+We don't have an end-to-end solution to this, yet.
+
+In the meantime, you can pull our production sandbox image
+(`docker.io/usercont/sandcastle:prod`) and run the command inside. As an
+example, this is how we were debugging build problems with anaconda:
+
+1. Clone the upstream git repo.
+
+2. Pull the sandbox image:
+
+```
+$ docker pull docker.io/usercont/sandcastle:prod
+```
+
+3. Launch the container and bind-mount the upstream project inside:
+
+```
+$ docker run -ti --rm -v $PWD:/anaconda docker.io/usercont/sandcastle:prod bash
+```
+
+4. Run commands of your choice:
+
+```
+[root@4af5dbd9c828 /]# cd /anaconda
+
+[root@4af5dbd9c828 anaconda]# ./configure
+checking for a BSD-compatible install... /usr/bin/install -c
+checking whether build environment is sane... yes
+checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
+checking for gawk... gawk
+checking whether make sets $(MAKE)... yes
+checking whether make supports nested variables... yes
+checking whether UID '0' is supported by ustar format... yes
+checking whether GID '0' is supported by ustar format... yes
+checking how to create a ustar tar archive... gnutar
+checking whether make supports nested variables... (cached) yes
+checking whether make supports the include directive... yes (GNU style)
+checking for gcc... gcc
+checking whether the C compiler works... yes
+...
+```
+
+Our deployment is running in [OpenShift
+Online](https://www.openshift.com/products/online/) which is using docker as a
+container engine, that's why we are using docker here and not
+[podman](https://github.com/containers/podman) - if you wanna try podman,
+please make sure to run it as root.
```

Comment:
I don't really understand why it have to be run as root when the output of this is just to test srpm creation in that environment.

Also to be able to run ``podman`` with ``-v`` you have to use ``-v $PWD:/anaconda:Z`` the :Z is mandatory otherwise selinux will kill the mount.

Reply:
thanks, I'm gonna add :Z, that's a good point

As for the root requirements: [rootless podman has its limitations](https://github.com/containers/podman/blob/master/rootless.md) and I'm personally tired off trying to figure those out or workaround them. Not even mentioning the performance hit b/c of the I/O being done in userspace via fuse - none of those happen when running as root.

Reply:
@TomasTomecek could you share more details on the performance impact of using rootless podman you are seeing? I [was not able to reproduce anything significant](https://github.com/packit/dist-git-to-source-git/pull/31#discussion_r476283708).

podman is meant to be used in rootless mode, so I think a recommendation from our side to run it as root should be well explained, and the reasons given should be concrete and quantifiable.

Reply:
Seems like I was wrong about the I/O: https://blog.tomecek.net/post/rootless-podman-io/

so I removed that part from the documentation